oracle/src/sender.js

```diff
@@ -30,7 +30,6 @@ if (process.argv.length < 3) {
 const config = require(path.join('../config/', process.argv[2]))
 
 const web3Instance = config.web3
-const nonceLock = `lock:${config.id}:nonce`
```

Comment:
Should we consider a special upgrade procedure if an existing oracle instance is upgraded?

Reply:
I don't think we need a special upgrade procedure. Now the sender won't check for the nonce lock key on redis, so even if the nonce was locked and the sender was stopped before the unlock, when the upgraded sender starts again, it won't be affected

Reply:
thanks